ar_exercise/exercise.rb

```diff
@@ -34,9 +34,17 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
```

Comment:
See the note-- don't comment out that stuff-- it will prevent ActiveRecord from loading